web/enoent.jsp

```diff
@@ -54,7 +54,7 @@ include file="menu.jspf"
 {
     PageConfig cfg = PageConfig.get(request);
     String configError = "";
-    if (cfg.getSourceRootPath().isEmpty()) {
+    if (cfg.getSourceRootPath() == null || cfg.getSourceRootPath().isEmpty()) {
```

Comment:
This `errorPage="error.jsp"` means that when there is an exception it will be passed to the error.jsp. This `isErrorPage="true"` means that this page can access the `exception` object. Using this in `enoent.jsp` means that any exception in `enoent.jsp` will be handled by `error.jsp` meanwhile `enoent.jsp` stays an error page for 404 errors.

The stuff in `enoent.jsp` is needed for the usual 404 (file not found) errors. Actually it should not be necessary for the "no configuration" error which is handled via the exception and `error.jsp`.